app/Http/Controllers/Bible/BooksControllerV2.php

```diff
@@ -411,26 +411,6 @@ public function chapters()
 
     private function getTestamentString($id)
```

Comment:
My plan in the future is to remove this method from controller for the ticket 235 to use only the helper method.